src/governor.cairo

```diff
@@ -405,11 +410,7 @@ pub mod Governor {
         }
 
         fn get_config_version(self: @ContractState, version: u64) -> Config {
```

Comment:
to rename `get_config`?  or `get_config_by_version` ?

it returns a Config, not a version, `get_config_version` is probably not the better name for this function?

Reply:
agreed, but slightly concerned about breaking compatibility with any calling contracts, and it's also confusing with `get_config_with_version` anyway